CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
   endif()
 
   option(USE_CLING "Use Cling as backend" OFF)
-  option(USE_REPL "Use clang-repl as backend" OFF)
+  option(USE_REPL "Use clang-repl as backend" ON)
```

Comment:
As the block below points out the following
```
  if (USE_CLING AND USE_REPL)
    message(FATAL_ERROR "We can only use Cling (USE_CLING=On) or Repl (USE_REPL=On), but not both of them.")
  endif()
```

What can be done if we are using REPL as default ....  is to make cling and REPL mutually exclusive. Which means we only use 1 of these by default.
1) REPL by default , cling off by default
2) And if cling is made on .... you can `set(USE_CLING ON)` and `unset(USE_REPL)` or anything related in cmake.

So that if we encounter something like (the case you've not addressed yet)

https://github.com/compiler-research/CppInterOp/blob/f794dee0a636b8c958927aa527440474c28de5b8/.github/workflows/ci.yml#L873-L874

We can just use USE_CLING and we don't need to explicitly set REPL to off.

Reply:
So we just end up with either the default case (which is supplying nothing, understood that repl is on and cling is off) or we just use cling=ON (understood repl would be off)